scripts/lib/mkcloud-driver-libvirt.sh

```diff
@@ -387,7 +387,7 @@ function libvirt_do_onhost_deploy_image()
             local part=$($sudo kpartx -asv $disk|perl -ne 'm/add map (\S+'"$last_part"') / && print $1')
             test -n "$part" || complain 31 "failed to find partition #$last_part"
             local bdev=/dev/mapper/$part
-            safely $sudo fsck -y -f $bdev
+            safely $sudo e2fsck -y -f $bdev
```

Comment:
could we also use `fsck -t ext4` ? might make it a bit more universal in case we have images with another rootfs later

Reply:
The '-t' argument for fsck is a selector by type for which devices to check, not for declaring the type of a device.